zipkin-server/src/main/java/zipkin2/server/internal/throttle/ThrottledCall.java

```diff
@@ -38,6 +40,13 @@
  * @see ThrottledStorageComponent
  */
 final class ThrottledCall extends Call.Base<Void> {
+  /**
+   * Rather than flooding when concurrency reached, return the same instance. The path to this is
+   * unimportant, so we clear the trace.
+   */
+  static final RejectedExecutionException STORAGE_THROTTLE_MAX_CONCURRENCY =
+    clearTrace(new RejectedExecutionException("STORAGE_THROTTLE_MAX_CONCURRENCY reached"));
```

Comment:
Maybe we can add a link to a javadoc or something to these trace-cleared exceptions to give users something to check if they've never seen the exception before.

Reply:
good idea